var/spack/repos/builtin/packages/crmc/package.py

```diff
@@ -50,6 +50,5 @@ def cmake_args(self):
         ]
         if self.spec.satisfies("@1.6.0:"):
             args.append("-D__HIJING__=ON")
-        if self.spec.satisfies("%gcc@9:"):

```

Comment:
Why was this conditional removed?

Reply:
Because clang is throwing type mismatch errors for scalar and rank1 arrays during build time

Reply:
I could add a condition for clang 14 instead of removing the condition completely though!